### DISCUSSION TITLE
GHANGELOG lacks of information about releases

### DISCUSSION THREAD

**Discussion by vasilich6107** [zino-app/graphql-flutter#1063] [Q&A] [answered]
Hi

I'm trying to understand what has changed between the version of the package.
I'm looking 
https://github.com/zino-hofmann/graphql-flutter/blob/main/packages/graphql/CHANGELOG.md
https://github.com/zino-hofmann/graphql-flutter/blob/beta/packages/graphql/CHANGELOG.md

Changelogs are bloated with duplicated items about `5.0.1-beta.1` release and the information about recent releases is missing.
Could you clarify is there any place to tack changes between releases?

**Comment by vincenzopalazzo** [comment, marked as answer]
Hi, sorry for the delay here!

Mh, I think there is something wrong with the changelog, and I'm sorry for that. We are working on a new tool to build the changelog easily https://github.com/vincenzopalazzo/changelog.dart

For the moment the big feature introduced is with the following PR

- https://github.com/zino-hofmann/graphql-flutter/pull/1034
- https://github.com/zino-hofmann/graphql-flutter/pull/1017
- https://github.com/zino-hofmann/graphql-flutter/pull/1016
- https://github.com/zino-hofmann/graphql-flutter/pull/1013
- https://github.com/zino-hofmann/graphql-flutter/pull/1051

I hope for the next release we will have a better changelog :) and thanks for pinging me out for this problem.

**Comment by vincenzopalazzo** [comment]
We fixed this by using https://github.com/vincenzopalazzo/changelog.dart
